talknet/demoTalkNet.py

```
@@ -1,726 +1,726 @@
+		frame = numpy.pad(image, ((bsi,bsi), (bsi,bsi), (0, 0)), 'constant', constant_values=(110, 110))
+		my  = dets['y'][fidx] + bsi  # BBox center Y
+		mx  = dets['x'][fidx] + bsi  # BBox center X
+		face = frame[int(my-bs):int(my+bs*(1+2*cs)),int(mx-bs*(1+cs)):int(mx+bs*(1+cs))]
+		vOut.write(cv2.resize(face, (224, 224)))
+	audioTmp    = cropFile + '.wav'
+	audioStart  = (track['frame'][0]) / 25
+	audioEnd    = (track['frame'][-1]+1) / 25
+	vOut.release()
+	command = ("ffmpeg -y -i %s -async 1 -ac 1 -vn -acodec pcm_s16le -ar 16000 -threads %d -ss %.3f -to %.3f %s -loglevel panic" % \
+		      (audioFilePath, data_loader_thread, audioStart, audioEnd, audioTmp)) 
+	output = subprocess.call(command, shell=True, stdout=None) # Crop audio file
```

Ruff (0.13.3)

`subprocess` call with `shell=True` identified, security issue

(S602)